src/app/components/Footer.tsx

```diff
@@ -19,49 +19,130 @@
 
 import { getAppConfig } from '@/global/config';
 import * as urls from '@/global/urls';
-import { Footer } from '@icgc-argo/uikit';
+import { css, useTheme } from '@/lib/emotion';
+import { Icon, Link } from '@icgc-argo/uikit';
+import Image from 'next/image';
+import { Col, Row } from 'react-grid-system';
+import rdpcLogo from '/public/assets/rdpc-logo.svg';
 
 const { UI_VERSION, REGION } = getAppConfig();
 const subtitle = `RDPC ${REGION} Clinical Data Submission Portal - ${UI_VERSION}`;
 
-const Logo = () => <div>RDPC</div>;
+const Logo = () => <Image src={rdpcLogo} alt="RDPC logo" />;
 
-export default function GlobalFooter() {
+const links = [
+	{
+		displayName: 'Contact',
+		href: '/contact',
+	},
+	{
+		displayName: 'Documentation',
+		href: urls.DOCS_URL_ROOT,
+		target: '_blank',
+	},
+	{
+		displayName: 'The Team',
+		href: '',
+	},
+	{
+		displayName: 'Privacy Policy',
+		href: urls.ARGO_PRIVACY_PAGE,
+		target: '_blank',
+	},
+	{
+		displayName: 'Terms & Conditions',
+		href: urls.ARGO_TERMS_PAGE,
+		target: '_blank',
+	},
+	{
+		displayName: 'Publication Policy',
+		href: urls.ARGO_PUBLICATION_PAGE,
+		target: '_blank',
+	},
+];
+
+export default function Footer() {
+	const theme = useTheme();
 	return (
-		<Footer
-			Logo={<Logo />}
-			subtitle={subtitle}
-			version={UI_VERSION}
-			links={[
-				{
-					displayName: 'Contact',
-					href: '/contact',
-				},
-				{
-					displayName: 'Documentation',
-					href: urls.DOCS_URL_ROOT,
-					target: '_blank',
-				},
-				{
-					displayName: 'The Team',
-					href: '',
-				},
-				{
-					displayName: 'Privacy Policy',
-					href: urls.ARGO_PRIVACY_PAGE,
-					target: '_blank',
-				},
-				{
-					displayName: 'Terms & Conditions',
-					href: urls.ARGO_TERMS_PAGE,
-					target: '_blank',
-				},
-				{
-					displayName: 'Publication Policy',
-					href: urls.ARGO_PUBLICATION_PAGE,
-					target: '_blank',
-				},
-			]}
-		/>
+		<footer
+			css={css`
+				font-size: 14px;
+				background: #fff;
+				z-index: 1;
+				padding: 0 14px 0 24px;
+				border-top: 1px solid ${theme.colors.grey_2};
+				font-size: 11px;
+				min-height: 58px;
+
+				& a + svg {
+					position: relative;
+					top: 3px;
+					margin: 0px 8px;
+				}
+			`}
+		>
+			<Row
+				css={css`
+					min-height: 58px;
+				`}
+				justify="between"
+				nogutter
+			>
+				<Col
+					md={5}
+					css={css`
+						display: flex;
+						align-items: center;
+					`}
+				>
+					{/** copyright and ARGO logo */}
+					<div>
+						© {new Date().getFullYear()} ICGC ARGO RDPC. All rights reserved.
+						<br />
+						{subtitle}
+					</div>
+				</Col>
+
+				<Col
+					md={7}
+					css={css`
+						display: flex;
+						align-items: center;
+						font-size: 12px;
+						justify-content: flex-end;
+						padding-left: 22px;
+					`}
+				>
+					{/** nav links */}
+					<div
+						css={css`
+							margin-right: 22px;
+						`}
+					>
+						{links.map(({ displayName, href, target }, idx) => (
```

Comment:
snark for comedic tone, not for snub:
how is idx better than index? "3 > 5" economy?